sdk/eventhubs/azure-messaging-eventhubs/src/main/java/com/azure/messaging/eventhubs/implementation/PartitionBasedLoadBalancer.java

```diff
@@ -93,17 +96,17 @@ public void loadBalance() {
         /*
          * Retrieve current partition ownership details from the datastore.
          */
-        final Mono<Map<String, PartitionOwnership>> partitionOwnershipMono = partitionManager
-            .listOwnership(eventHubName, consumerGroupName)
-            .timeout(Duration.ofSeconds(1)) // TODO: configurable by the user
+        final Mono<Map<String, PartitionOwnership>> partitionOwnershipMono = eventProcessorStore
+            .listOwnership(fullyQualifiedNamespace, eventHubName, consumerGroupName)
+            .timeout(Duration.ofSeconds(2)) // TODO: configurable by the user
```

Comment:
I guess the timeout in retry options is used for receiving events from Event Hubs. This timeout is for requests to the store (Blob storage). Since they may require different timeout durations, I wanted to keep them separate.